### PR TITLE
Deleting a droplet

### PR DESCRIPTION
**Pull request by Pritom14**

### CODE REVIEW

**Comment by Pritom14** [comment]
Sir @championswimmer ,please review the PR
18597173_1781136385531797_1922954813_o

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java [hunk 1]
```diff
@@ -87,8 +89,34 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
         if (id == R.id.delete_droplet) {
-
             // TODO: 26/11/16 perform delete
+            new MaterialDialog.Builder(this)
+                    .title(R.string.delete_droplet)
+                    .content("Are you sure you would like to permanently delete "+String.valueOf(droplet.getName())+" ?")
```
Comment (championswimmer) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java:95]: into strings.xml

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java [hunk 2]
```diff
@@ -87,8 +89,34 @@ public boolean onCreateOptionsMenu(Menu menu) {
...
+                    .positiveText("Confirm").onPositive(new MaterialDialog.SingleButtonCallback() {
+                @Override
+                public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which) {
+                    doaClient.performAction(droplet.getId(),ActionType.DESTROY,null).enqueue(new Callback<Action>() {
+                        @Override
+                        public void onResponse(Call<Action> call, Response<Action> response) {
+                            Log.d("DESTROY",String.valueOf(response.code()));
```
Comment (championswimmer) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java:102]: set global tag name of class

**Review by championswimmer — CHANGES_REQUESTED** [review]

**Comment by championswimmer** [comment]
@Pritom14 please make the noted changes if you're still interested in working on this

**Comment by utkarshmttl** [comment]
Can I make thw requested changes and submit another PR? @championswimmer

**Comment by championswimmer** [comment]
stagnant

**Pull request closed without merging**